.github/workflows/main.yml

```diff
@@ -0,0 +1,118 @@
+name: CI
```

Comment:
The azure version has a filename and top-level name that mentions "linux ci" ... is that useful to capture? Or hard?

Reply:
I am going to merge all three files to one. Starting with linux. Once they are merged the platform names will display in the runs below.